forwarder/config_type.go

```diff
@@ -54,17 +54,11 @@ type ForwarderHttpApiConfig struct {
 }
 
 type ForwarderGatewayConfig struct {
-	// Onboarder configures the gateway onboarder smart contract plugin.
-	Onboarder struct {
-		// Address is the smart contract chain address
-		Address common.Address `mapstructure:"address"`
-	} `mapstructure:"onboarder"`
-
 	// BatchOnboarder configures the gateway batch onboarder smart contract plugin.
 	BatchOnboarder struct {
 		// Address is the smart contract chain address
 		Address common.Address `mapstructure:"address"`
-	} `mapstructure:"batch_onboarder"`
+	} `mapstructure:"onboarder"`
```

Comment:
If we do this, we should also rename the variable I feel.

Reply:
That was intentional to make it in the code clear that the forwarder uses the batch onboard plugin. If you want to have config and field names match I prefer to switch the config name back to `batch_onboarder` instead of renaming the field.

Reply:
Ok then I would propose to rename it back